packages/manager/apps/public-cloud/src/components/easter-eggs/modal/controller.js

```diff
@@ -0,0 +1,18 @@
+export default class EeasterEggsModalController {
+  /* @ngInject */
+  constructor($uibModalInstance) {
+    this.$uibModalInstance = $uibModalInstance;
+  }
+
+  activate() {
+    return this.cancel(true);
+  }
+
+  cancel(result = null) {
+    return this.$uibModalInstance.close(result);
```

Comment:
Like that ^.^

Reply:
I'm not able to reach this anchor. Is the link is the correct one?